addons/main/functions/fnc_drawDownedUnitIndicator.sqf

```diff
@@ -8,10 +8,17 @@ private _color = [
     profileNamespace getvariable ['igui_error_RGB_B', 0.1],
     profileNamespace getvariable ['igui_error_RGB_A', 1]
 ];
+private _isMedic = player getUnitTrait "Medic";
```

Comment:
We want it to be zeus compatible, so in case zeus possesses a unit, we gotta take that into account. This cba functions returns the unit the player is currently in control of.

```suggestion
private _isMedic = (call CBA_fnc_currentUnit) getUnitTrait "Medic";
```